test_scripts/Smoke/API/025_UnsubscribeVehicleData_PositiveCase.lua

```diff
@@ -0,0 +1,152 @@
+  fuelLevel = "VEHICLEDATA_FUELLEVEL",
+  fuelLevel_State = "VEHICLEDATA_FUELLEVEL_STATE",
+  instantFuelConsumption = "VEHICLEDATA_FUELCONSUMPTION",
+  externalTemperature = "VEHICLEDATA_EXTERNTEMP",
+  prndl = "VEHICLEDATA_PRNDL",
+  tirePressure = "VEHICLEDATA_TIREPRESSURE",
+  odometer = "VEHICLEDATA_ODOMETER",
+  beltStatus = "VEHICLEDATA_BELTSTATUS",
+  bodyInformation = "VEHICLEDATA_BODYINFO",
+  deviceStatus = "VEHICLEDATA_DEVICESTATUS",
+  driverBraking = "VEHICLEDATA_BRAKING",
+  wiperStatus = "VEHICLEDATA_WIPERSTATUS",
+  headLampStatus = "VEHICLEDATA_HEADLAMPSTATUS",
+  engineTorque = "VEHICLEDATA_ENGINETORQUE",
+  accPedalPosition = "VEHICLEDATA_ACCPEDAL",
+  steeringWheelAngle = "VEHICLEDATA_STEERINGWHEEL",
+  eCallInfo = "VEHICLEDATA_ECALLINFO",
+  airbagStatus = "VEHICLEDATA_AIRBAGSTATUS",
+  emergencyEvent = "VEHICLEDATA_EMERGENCYEVENT",
+  clusterModeStatus = "VEHICLEDATA_CLUSTERMODESTATUS",
+  myKey = "VEHICLEDATA_MYKEY"
+}
+
+local paramsList = {}
+local requestParams = { }
+for k, _ in pairs(VDValues) do
+  table.insert(paramsList, k)
+  requestParams[k] = true
+end
+
+local responseUiParams = { }
+
+local allParams = {
+  requestParams = requestParams,
+  responseUiParams = responseUiParams,
+}
+
+--[[ Local Functions ]]
+local function ptuUpdateFunc(tbl)
+  local SVDgroup = {
+    rpcs = {
+      SubscribeVehicleData = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
+        parameters = paramsList
+      },
+      UnsubscribeVehicleData = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
+        parameters = paramsList
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings.NewTestCaseGroup = SVDgroup
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups =
+  { "Base-4", "NewTestCaseGroup" }
+end
+
+local function setVDResponse()
+  local temp = { }
+  local vehicleDataResultCodeValue = "SUCCESS"
+  for k, v in pairs(VDValues) do
+    if "clusterModeStatus" == k then
+      temp["clusterModes"] = {
+        resultCode = vehicleDataResultCodeValue,
+        dataType = v
+      }
+    else
+      temp[k] = {
+        resultCode = vehicleDataResultCodeValue,
+        dataType = v
+      }
+    end
+  end
+  return temp
+end
+
+local function subscribeVD(pParams, self)
+  local cid = self.mobileSession1:SendRPC("SubscribeVehicleData", pParams.requestParams)
+  pParams.responseUiParams = setVDResponse()
+  EXPECT_HMICALL("VehicleInfo.SubscribeVehicleData", pParams.requestParams)
+  :Do(function(_, data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", pParams.responseUiParams)
+    end)
+  self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  self.mobileSession1:ExpectNotification("OnHashChange")
+end
+
+local function unsubscribeVD(pParams, self)
+  local cid = self.mobileSession1:SendRPC("UnsubscribeVehicleData", pParams.requestParams)
+  pParams.responseUiParams = setVDResponse()
```

Comment:
@GetmanetsIrina probably this line is redundant because responseUiParams will be filled up in `subscribeVD` function at previous step

Reply:
@AKalinich-Luxoft , find update related to all comments in d5d3d3d0a890b0291e8dddbf8266d6150c081b01